NEWS.md

```diff
@@ -1,4 +1,4 @@
-# datawizard 0.11.0.1
+# datawizard 0.11.1
```

Comment:
can you keep the 4th digit as in the DESCRIPTION file? This makes it easier to see that it's unreleased